qml/Stage/Stage.qml

```diff
@@ -1766,7 +1767,17 @@ FocusScope {
             }
         }
 
-        onPressed: mouse.accepted = false
+        onPressed: {
+          mouse.accepted = false;
+          if (mouse.source === Qt.MouseEventSynthesizedByQt)
+              isTouch = true;
+          else
+              isTouch = false;
```

Comment:
Could we get this down to a single line `isTouch = mouse.source === Qt.Mouse...` just to make it a bit cleaner

Reply:
Yeah sorry about that, had it like this since i had some console.log's in there to debug with.